zzre/game/StdSpellId.cs

```diff
@@ -25,12 +25,12 @@ public static class StdSpells
     private static bool IsSpellCompatible(SpellRow spell, zzio.ZZClass zzClass, uint level)
     {
         var price =
-            (spell.PriceA > 0 ? 1 : 0) +
-            (spell.PriceB > 1 ? 1 : 0) +
-            (spell.PriceC > 1 ? 1 : 0);
+            (spell.PriceA != zzio.ZZClass.None ? 1 : 0) +
+            (spell.PriceB != zzio.ZZClass.None && spell.PriceB != zzio.ZZClass.Nature ? 1 : 0) +
+            (spell.PriceC != zzio.ZZClass.None && spell.PriceC != zzio.ZZClass.Nature ? 1 : 0);
```

Comment:
sorry again about that method, the second term is not necessary either, just the ZZClass.None check for all three slots